pulp_python/app/models.py

```diff
@@ -105,19 +109,23 @@ class PythonImporter(Importer):
     TYPE = 'python'
     projects = models.TextField(null=True)
 
-    def _fetch_inventory(self):
+    def _fetch_inventory(self, old_version):
         """
         Fetch existing contentunits in the repository.
 
+        Args:
+           old_version (pulpcore.plugin.models.RepositoryVersion): the latest pre-existing version
```

Comment:
I don't think this should be called ```old_version``` if it is the latest existing version.  Maybe ```current_version``` (as opposed to "new_version" or something).